packages/flutter/test/material/about_test.dart

```diff
@@ -565,9 +580,12 @@ void main() {
       tester.getTopLeft(find.text('Licenses')),
       const Offset(16.0 + safeareaPadding, 14.0 + safeareaPadding),
     );
-  }, skip: kIsWeb && !isSkiaWeb); // https://github.com/flutter/flutter/issues/99933
+  },
+      skip: kIsWeb &&
```

Comment:
Looks like this might have been broken into multiple lines due to a formatter. The linter is looking for the issue for the skip is on the same line, so if this is reverted it should fix the broken check on the PR.